tests/helpers.py

```diff
@@ -1,19 +1,7 @@
-import os
 import subprocess
-import time
 
-def tree_correct(path):
-    with open("/tmp/output.txt", "wb") as out, open("/tmp/err.txt", "wb") as err:
-        subprocess.Popen(["tree",path], stdout=out, stderr=err)
-    time.sleep(1)
-    with open('/tmp/output.txt') as result:
-        output = result.read()
-    return output
 
-def pytree(path):
-    with open("/tmp/pytree.txt", "wb") as out, open("/tmp/pyerr.txt", "wb") as err:
-        subprocess.Popen(["./pytree.py",path], stdout=out, stderr=err)
-    time.sleep(1)
-    with open('/tmp/pytree.txt') as result:
-        output = result.read()
-    return output
+def run_and_capture(cmd, path):
+    result = subprocess.run([cmd, path], stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
```

Comment:
The reason I piped out to file in /tmp/ was to capture the output and allow students to explore the differences. While Pytest does provide a diff when two strings are compared. There are several small errors such as utf symbols, which must be used for an exact match.